Project.toml

```diff
@@ -47,8 +49,9 @@ StaticArrays = "0.12, 1.0"
 julia = "1.6"
 
 [extras]
+ApproxFunOrthogonalPolynomials = "b70543e2-c0d9-56b8-a290-0d4d6d4de211"
```

Comment:
`ApproxFunOrthogonalPolynomials` i probably not needed anymore, as tests are moved to that repo?

Reply:
changed